### ISSUE TITLE
[Input][A11y]: Clear button is not navigable from the keyboard, type button is not communicated to the user

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11018]
### Bug Description

**Observed Behavior:**
Switch `showClearIcon` to true, type something in the input field.
https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/inputs-input--default&args=showClearIcon:!true

The 'Clear' button in the input field is not accessible via keyboard navigation, and its type button is not announced by screen readers.

### Affected Component

Input

### Expected Behaviour

Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs must be accessible with keyboard.
With screen reader enabled, UI elements type for all the UIs should be communicated to the user.

### Priority

None

### UI5 Web Components Version

2.7.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rl   ,
could you please check this issue?
The issue reproduces following the described steps.

Thanks,
Todor

**Comment by ivoplashkov** [comment]
Hell @AlekseiSch 

Thank you for bringing this issue to our attention. However, after reviewing the accessibility behavior, we can confirm that it is not inaccessible but rather designed intentionally to be non-focusable.

The clear icon is marked as decorative with role=presentation, meaning it does not receive focus and is not announced by screen readers as an interactive element. This is because its function - clearing the input field, can be achieved through standard keyboard interactions, making it redundant to provide an additional focusable control.

For screen reader users, the clear icon can still be accessed via the Virtual Cursor. Additionally, the same functionality can be achieved using the keyboard by pressing Ctrl/Cmd + A to select the entire input value and then pressing Backspace to clear it.

Since the expected behavior aligns with accessibility best practices, this does not appear to be an issue but rather an intentional design choice.

Best regards,
Ivaylo

**Comment by AlekseiSch** [comment]
Hello @ivoplashkov 
Thank you for your detailed response. However, I’d like to ask to reopen the issue, as marking the Clear Icon as decorative and non-focusable may not align with accessibility guidelines or best practices.
According to the recommendations https://www.w3.org/TR/wai-aria-1.1/#presentation

> The intended use is when an element is used to change the look of the page but does **not have all the functional, interactive**, or structural relevance implied by the element type

> Example use cases:
An element whose content is completely presentational (like a spacer image, decorative graphic, or clearing element);
An image that is in a container with the img role and where the full text alternative is available and is marked up with aria-labelledby and (if needed) aria-describedby;
An element used as an additional markup "hook" for CSS; or
A layout table and/or any of its associated rows, cells, etc.

Accessibility standards describe that keyboard navigation must be available for **all interactive elements**. The Clear Icon provides functionality, and users will expect to interact with it via the keyboard. The lack of focusability for this control violates WCAG 2.1 principles https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

If the Clear Icon is not interactive, it introduces a mismatch between visual and functional behaviors, violating WCAG principles.

Thank you for reconsidering this issue.
Best regards,
Aleksei
